### DISCUSSION TITLE
Sim2real, reality gap

### DISCUSSION THREAD

**Discussion by ndolphin-github** [SofaDefrost/SoftRobots#275] [Debate - Technical topics]
Hi,
I am currently using SOFA to transition my custom-built soft robot model into a simulation environment. I have a question regarding the sim2real process.

How can I adjust the scripts in SOFA's Python environment to minimize the reality gap between the simulation model and the actual measured data from the robot? Are there any well-documented methods for setting up SOFA simulations for this purpose?

I am using the latest version of SOFA, v23.06, along with Python 3 libraries. To my knowledge, there are several parameters I can edit, such as the Young's modulus or the Poisson ratio of the material. However, I am uncertain if the reality gap can be significantly reduced by merely adjusting some parameters. Are there any recommended publications that detail the process of sim2real using SOFA?

**Comment by TeoREN0217** [comment]
https://ieeexplore.ieee.org/abstract/document/10171986
